reference/resources/hostendpoint.md

```diff
@@ -21,17 +21,22 @@ will use to apply
 [policy]({{ site.baseurl }}/reference/resources/networkpolicy)
 to the interface.
 
+> **Important**: When rendering security rules on other hosts, {{site.prodname}} uses the
+> `expectedIPs` field to resolve label selectors to IP addresses. If the `expectedIPs` field
+> is omitted then security rules that use labels will fail to match this endpoint.
+{: .alert .alert-danger}
+
+> **Note**: Host endpoints with `interfaceName: *` do not support [untracked policy]({{ site.baseurl }}/security/high-connection-workloads).
+{: .alert .alert-info}
+
 For `calicoctl` [commands]({{ site.baseurl }}/reference/calicoctl/overview) that specify a resource type on the CLI, the following
 aliases are supported (all case insensitive): `hostendpoint`, `hostendpoints`, `hep`, `heps`.
 
-**Default behavior of external traffic to/from host**
+### Default behavior of external traffic to/from host
 
 If a host endpoint is created and network policy is not in place, the {{site.prodname}} default is to deny traffic to/from that endpoint (except for traffic allowed by failsafe rules).
```

Comment:
This isn't true for the auto-host endpoints is it, or more correctly depends on the profile field in the HEP configuration.

Reply:
I guess you could argue the profile applies some network policy - but I think it make sense to spell this out.

Reply:
I see what you mean but there is a blurb about the exception for auto host endpoints a bit further down, after `However, profiles can be used in conjunction...`.

Maybe I can improve that line:
> Auto host endpoints specify the `projectcalico-default-allow` profile so they behave similarly to pod workload endpoints. In the absence of network policy, auto host endpoints allow all traffic through them.

WDYT?

Reply:
To me it feels like it contradicts itself.  "If a host endpoint is created" doesn't imply "without a profile" IMHO.  Perhaps just some minor tweaks and a reorder....   so spell out the "without a profile" and change "deny" -> "police" for the paras about named and wildcard.  I dunno, I'm terrible at this kind of thing.  WDYT?

If a host endpoint is created without a profile, and network policy is not in place, the {{site.prodname}} default is to deny traffic to/from that endpoint (except for traffic allowed by failsafe rules).

Profiles can be used in conjunction with host endpoints to modify default behavior of external traffic to/from the host in the absence of network policy.  {{site.prodname}} defines a profile `projectcalico-default-allow` used to change the default to allow.

> Auto host endpoints specify the `projectcalico-default-allow` profile so they behave similarly to pod workload endpoints.
 {: .alert .alert-info}

For a named host endpoint (i.e. a host endpoint representing a specific interface), {{site.prodname}} polices traffic only to/from the interface specified in the host endpoint (except for traffic allowed by failsafe rules). Traffic to/from other interfaces is ignored.

 For a wildcard host endpoint (i.e. a host endpoint representing all of a host's interfaces), {{site.prodname}} polices traffic to/from _all_ interfaces on the host (except for traffic allowed by failsafe rules).